src/layer/AbstractSentinelHubV3Layer.ts

```diff
@@ -537,4 +674,11 @@ export class AbstractSentinelHubV3Layer extends AbstractLayer {
     const shServiceHostname = this.getShServiceHostname();
     return `${shServiceHostname}api/v1/process/convertscript?datasetType=${this.dataset.shProcessingApiDatasourceAbbreviation}`;
   }
+
+  protected getFindTilesAdditionalParameters(): FindTilesAdditionalParameters {
+    return {
+      maxCloudCoverPercent: null,
+      datasetParameters: null,
+    };
+  }
```

Comment:
Since this is on `AbstractSentinelHubV3Layer`, it is used by all the child layers, including S-1 on AWS, where cloud cover doesn't apply - I think S-1 shouldn't know about CC.

I think it might make sense to redefine `FindTilesAdditionalParameters` so that it can include any parameters (`Record<string, any>`) and let the child decide which ones to use. We can also define `FindTilesAdditionalParametersCC` if we want to be stricter with the keys, and then let layers decide what to use.

Reply:
Type `FindTilesAdditionalParameters` was changed to `Record<string, any>`